### PR TITLE
add admin auth and search filter for ldap

### PR DESCRIPTION
**Pull request by aizerin**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing applications:)

**What changes did you make?** (Give an overview)

I need more params to configure LDAP connection. If is it ok, I will add them to the docs and example.

**How Has This Been Tested?** (put an "X" next to an item)

- [ ] No need to
- [X] Manually(please, describe, when necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "X" next to an item, otherwise PR will fail)
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [ ] My changes generate no new warnings(e.g. Sonar is happy)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

### CODE REVIEW

**Comment by Haarolean** [comment]
Hey, thanks for your contribution! Please ping me once it's ready :)

**Comment by aizerin** [comment]
@Haarolean it is ready

File: docker/auth-ldap.yaml [hunk 1]
```diff
@@ -30,6 +30,38 @@ services:
...
+      KAFKA_CLUSTERS_1_NAME: secondLocal
+      KAFKA_CLUSTERS_1_BOOTSTRAPSERVERS: kafka1:29092
+      KAFKA_CLUSTERS_1_ZOOKEEPER: zookeeper1:2181
+      KAFKA_CLUSTERS_1_JMXPORT: 9998
+      KAFKA_CLUSTERS_1_SCHEMAREGISTRY: http://schemaregistry1:8085
+      KAFKA_CLUSTERS_1_KAFKACONNECT_0_NAME: first
+      KAFKA_CLUSTERS_1_KAFKACONNECT_0_ADDRESS: http://kafka-connect0:8083
+      AUTH_TYPE: "LDAP"
+      SPRING_LDAP_URLS: "ldap://ldap:10389"
+      SPRING_LDAP_USERFILTER_SEARCHBASE: "dc=planetexpress,dc=com"
```
Comment (Haarolean) [inline docker/auth-ldap.yaml:59]: I'd suggest we move these properties to the main container's env. variables and comment them out and remove this one.

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (Haarolean) [inline yarn.lock:1]: Can you rollback the changes to unrelated files please?

**Comment by Haarolean** [comment]
> @Haarolean it is ready

okay, thanks!

**Review by Haarolean — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 2 above)
Comment (aizerin) [reply yarn.lock]: done, I overlooked it.

File: docker/auth-ldap.yaml (hunk 1 above)
Comment (aizerin) [reply docker/auth-ldap.yaml]: done

**Review by germanosin — APPROVED** [review]
LGTM!

**Review by Haarolean — APPROVED** [review]
Thanks for your contribution!
